prebuilt-ui/basic-embed/README.md

```diff
@@ -0,0 +1,24 @@
+# Daily Prebuilt: Next.js demo 
+
+![Clicking create room button starts a call](./basic-embed.gif)
+
+## How the demo works 
+
+This demo embeds [Daily Prebuilt](https://www.daily.co/prebuilt), a ready-to-use video chat interface, into a Next.js site. It makes use of [Next API routes](https://nextjs.org/docs/api-routes/introduction) to create a Daily room server-side.  
+
+## Requirements
+
```

Comment:
Not for this readme but so I don't forget: It looks like there's _another_ readme in `/prebuilt` that lists what's include. Can you add this demo to that readme too please? (There's already an example for the webview one there)

Reply:
Great catch, thank you!